client/src/plugins/version-info/VersionInfoOverlay.js

```diff
@@ -14,7 +14,7 @@ import { Overlay, Section } from '../../shared/ui';
 
 import { ReleaseInfo } from './ReleaseInfo';
 
-const RELEASE_NOTES_LINK = 'https://camunda.com/blog/category/modeling/?tag=camunda-modeler&release-note&utm_source=modeler&utm_medium=referral';
+const RELEASE_NOTES_LINK = 'https://camunda.com/blog/category/releases/?s=desktop+modeler';
```

Comment:
Let's not remove marketing query params:
```suggestion
https://camunda.com/blog/category/releases/?s=desktop+modeler&utm_source=modeler&utm_medium=referral
```

Reply:
I will add this back